Gemfile

```diff
@@ -7,20 +7,20 @@ gemspec
 group :development, :test do
   gem "activesupport"
   gem "bundler", ">= 1.9.5"
+  gem "colored"
   gem "docker-api"
+  gem "dogstatsd-ruby", ">= 3.0.0", "< 4.7.0"
```

Comment:
I had to lower the maximum version number because after 4.7.0 there was a breaking change in this library that was causing errors, for more info on the error https://github.com/DataDog/dogstatsd-ruby/issues/134. To upgrade the version some changes (mainly not using the deprecated default values) are required in lib/kafka/datadog.rb. But these changes should be introduced in a different PR to avoid having too many changes here in my opinion.

Reply:
also we do not use datadog